doc/source/installation.rst

```diff
@@ -148,7 +223,7 @@ You can pass any additional compiler flags for the C/C++ compiler using the ``ex
 This allows you to add any additional compiler options required for your architecture.
 
 For example, ``extra_cflags`` can be used to tune your MDAnalysis installation for your current architecture using the `-march`, `-mtune`, `-mcpu` and related compiler flags.
-*Which* particular compiler flags to use depends on your CPU architecture. An example for an x86_64 machine would be to change the line in `setup.cfg` as follows:
+The optimal compiler flags depend on your CPU architecture. An example for an x86_64 machine would be to change the line in `setup.cfg` as follows:
```

Comment:
I don't think this suggestion should be limited to `x86_64`. ` -march=native -mtune=native` should work for different architectures. Maybe it was intended to use something like `-march=x86-64-v4`? Is there a better example we can use?